File: back/vidaplena/vidaplena/src/main/java/com/vidaplena/vidaplena/aplicacion/encuesta/dto/Encuesta.java
package com.vidaplena.vidaplena.aplicacion.encuesta.dto;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;
import org.springframework.stereotype.Repository;

import java.sql.Date;

@Getter
@Setter
@NoArgsConstructor
@ToString
@Repository
public class Encuesta {
    private Long idEncuesta;
    private String respuesta1;
    private String respuesta2;
    private String respuesta3;
    private String calificacion;
    private Date fechaCreacion;
}

File: src/app/model/encuesta.ts
import { Validators } from "@angular/forms";

export interface Encuesta {
    respuesta1: string;
    respuesta2: string;
    respuesta3: string;
    calificacion: string;
}

export const ENCUESTAFORM = {
    respuesta1: ['', Validators.required],
    respuesta2: ['', Validators.required],
    respuesta3: ['', Validators.required],
    calificacion: ['', Validators.required],
}